src/test/mediapipeflow_test.cpp

```diff
@@ -2840,13 +2857,68 @@ class MediapipeFlowStartTest : public TestWithTempDir {
         }
         ovms::Server& server = ovms::Server::instance();
         server.setShutdownRequest(0);
+        std::promise<void>().swap(unblockLoading2ndGraph);
```

Comment:
why do we need it? what does it do? why wasnt it required earlier?

Reply:
There was only 1 test run and I have added more. The unblockLoading2ndGraph is a global variable thus needs to be reseted for consecutive tests execution.